front/lib/labs/config.ts

```diff
@@ -7,11 +8,17 @@ const config = {
   getNangoSecretKey: (): string => {
     return EnvironmentConfig.getEnvVariable("NANGO_SECRET_KEY");
   },
-  getNangoGoogleDriveConnectorId: (): string => {
-    return EnvironmentConfig.getEnvVariable("NANGO_GOOGLE_DRIVE_CONNECTOR_ID");
-  },
-  getNangoGongConnectorId: (): string => {
-    return EnvironmentConfig.getEnvVariable("NANGO_GONG_CONNECTOR_ID");
+  getNangoConnectorIdForProvider: (provider: LabsConnectorProvider): string => {
```

Comment:
In the config let's avoid putting logic like this and do one getter per provider.

Reply:
Why? Is this a coding style thing on our side?
Makes things more complicated in places like this where I'd have to duplicate line, when typescript handles things nicely for now : https://github.com/dust-tt/dust/pull/5299/files#diff-4f152a814a0e5d1f3e1727fdd4f21be52e3f0f7a49bd5d4f49d912aebe114ea9R158

Reply:
Ok not strongly opinionated against this, it was mainly to have a clear one to one relationship with the environment variables.